### PR TITLE
Fix eslint warnnings

### PR DESCRIPTION
**Pull request by ericlowturing**
[Associated Notion Card](link here)

-----

## 1. Pull request checklist

- [ ] Notion card has been moved to the Code Review column
- [ ] Notion card has a link to this PR
- [ ] A reviewer has been assigned to the Notion card

### CODE REVIEW

File: src/containers/other/NFTView/NFTView.tsx [hunk 1]
```diff
@@ -411,10 +409,11 @@ const NFTView: React.FC<NFTViewProps> = ({ onTransfer }) => {
 		return (
 			<>
 				<div className={styles.Stats}>
-					{data.map((item) => (
+					{data.map((item, index) => (
 						<>
 							{!item.isHidden ? (
 								<StatsWidget
+									key={index}
```
Comment (colbr) [inline src/containers/other/NFTView/NFTView.tsx:416]: Use something other `index` as the `key` here - [iirc it's bad practice to use index](https://stackoverflow.com/questions/46735483/error-do-not-use-array-index-in-keys). I don't think we would run into problems (currently) by using `index` here, but best to change it in my opinion.

**Review by colbr — CHANGES_REQUESTED** [review]
One small change to make. Other than that, looking good! Thanks @ericlowturing

File: src/containers/other/NFTView/NFTView.tsx (hunk 1 above)
Comment (ericlowturing) [reply src/containers/other/NFTView/NFTView.tsx]: Agreed! Just updated the key so that we won't have redundant key value. Thanks

**Review by colbr — APPROVED** [review]
Nice!

**Pull request merged**